deploy/crds/storageos_v1_storageoscluster_cr.yaml

```diff
@@ -38,7 +38,7 @@ spec:
   #   DriverRequiresAttachment: "true"
   #   deploymentStrategy: "deployment"
   # service:
-  #   name: "storageos"
+  #   name: "kube-system"
```

Comment:
This shouldn't change. It's the storageos service name.

Reply:
good catch!